src-java/northbound-service/northbound/src/main/java/org/openkilda/northbound/controller/v2/YFlowControllerV2.java

```diff
@@ -151,4 +163,79 @@ public CompletableFuture<YFlowPingResult> pingYFlow(
     public CompletableFuture<YFlow> swapYFlowPaths(@PathVariable("y_flow_id") String yFlowId) {
         return flowService.swapYFlowPaths(yFlowId);
     }
+
+    private void verifyRequest(YFlowCreateUpdatePayloadBase request) {
```

Comment:
Here and below: all the methods below can be static. Overall, I think that you need separate FlowValidator class, where you can move all the validations from controller.

Reply:
Verify and convert input data is responsibility of controller. That why these validation code are located here.

Reply:
IIRC, nobody forbids you to delegate some class functions as long as you follow single responsibility principle. 
I recommend you to move validation to another class because these static methods appear to be very non-suitable here (at least for me). Especially when we already have Validator classes in code (e.g. `java/org/openkilda/wfm/topology/flowhs/validation/FlowValidator.java`)